javascript/javascript1/week2/Students

```diff
@@ -0,0 +1,32 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (class07Students.includes(studentName)) {
+        console.log(` ${studentName} is already in the class!`);
+        return;
+    }
+    if (class07Students.length >= 6 && studentName !== "Queen") {
```

Comment:
You already made a function to calculate the number of students. How would you reuse it here?